recipes/stress-ng/stress-ng.inc

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "stress-ng tests a system by stressing the physical and kernel interfaces."
+
+DESCRIPTION = "stress-ng will stress test a computer system in various selectable ways. It was designed to exercise various physical subsystems of a computer as well as the various operating system kernel interfaces."
+
+LICENSE = "GPLv2"
```

Comment:
Thanks. I somehow always end up using the GPLv2 since that seems more natural to me, but I'll have to unlearn that. I've updated it (and reading the source files more carefully, it's actually GPL-2.0+).